app/views/spree/amazon/address.html.erb

```diff
@@ -26,9 +26,13 @@
       new OffAmazonPayments.Widgets.AddressBook({
         sellerId: '<%= @amazon_gateway.preferred_merchant_id %>',
         agreementType: 'orderreference',
-        onOrderReferenceCreate: function(orderReference) {
-          order_reference = orderReference.getAmazonOrderReferenceId();
-        },
+          onOrderReferenceCreate: function(orderReference) {
```

Comment:
I'm not sure what is going on here. It looks like you added code but then commented it out. Was this incomplete?